recipes/trogon/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "trogon" %}
+{% set version = "0.5.1dev0" %}
```

Comment:
You should keep the stable version and add the license b/c if you choose the dev release way you must comply with https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Not 100% sure I follow what stable version you mean. I used `version` here for the name of the version that's on conda-forge rather than the sha. I could revert [981be4f](https://github.com/conda-forge/staged-recipes/pull/23391/commits/9e1be4f2b85a79a2e0b7db677a8c06d3c6f0d729) if that's what you mean. I followed the link you describe (particularly the appendix) for setting it as `dev0` which should get sorted correctly (the next release of 0.5.1 or 0.6.0 should sort above this version).

Reply:
`0.5.1dev0` is not a stable version. If you want to keep that you must follow https://github.com/conda-forge/cfep/blob/main/cfep-05.md.

The alternative is to use the stable release on PyPI, https://pypi.org/project/trogon/0.5.0/, and just add the license.

Reply:
How do I add the LICENSE to the stable release on pypi?

Reply:
You download it, place in the recipe directory, and list it in the meta.yaml.